src/configurations/messages.js

```diff
@@ -6,6 +6,9 @@ const messagingConfig = {
             unknown_error: 'Look\'s like there is a problem in our side. Please try again later.',
             already_exists_error: 'Look\'s like this {0} is already taken. Please try again.',
             non_authenticated_error: 'You need to be logged in to proceed ahead.',
+            empty_field_error: 'The {} field cannot be empty. Please try again.',
+            empty_otp_field: 'Please enter otp',
```

Comment:
@vikas339 You can use `empty_field_error` for the `empty_otp_field` cases. Keep it as generic as possible.